packages/itwinui-css/src/button/base.scss

```diff
@@ -12,7 +12,6 @@
   justify-content: center;
   position: relative;
   border-radius: var(--iui-border-radius-1);
-  line-height: 1.2;
```

Comment:
Might need to revert this, some buttons (but not all) look like the label & icon are not vertically centered after this change.

Reply:
Reverted.